### PR TITLE
Tutorial rework per issue #688

### PR DESCRIPTION
**Pull request by intelkevinputnam**
Reworked tutorial section and moved cloud install content to get started section.

Signed-off-by: Kevin Putnam <kevin.putnam@intel.com>

### CODE REVIEW

File: source/tutorials/docker/docker.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 .. _docker:
 
-Run Docker\* on |CL-ATTR|
-#########################
+Run Docker\*
```
Comment (puneetse) [inline source/tutorials/docker/docker.rst]:
```suggestion
Docker\*
```

File: source/tutorials/proxy.rst [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .. _tutorial-proxy:
 
-Setting up proxy
-################
+Proxy
```
Comment (puneetse) [inline source/tutorials/proxy.rst]:
```suggestion
Proxy configuration
```

File: source/tutorials/yubikey-u2f.rst [hunk 3]
```diff
@@ -1,7 +1,7 @@
 .. _yubikey-u2f:
 
-Enable YubiKey U2F Support 
-##########################
+YubiKey\* U2F Support
```
Comment (puneetse) [inline source/tutorials/yubikey-u2f.rst]:
```suggestion
YubiKey\* Support 
```

File: source/tutorials/fmv.rst [hunk 4]
```diff
@@ -1,7 +1,7 @@
 .. _fmv:
 
-Use the function multi-version patch generator
-##############################################
+Function multi-version patch generator
```
Comment (puneetse) [inline source/tutorials/fmv.rst]:
```suggestion
Function multi-versioning
```

File: source/tutorials/index.rst [hunk 5]
```diff
@@ -13,17 +13,15 @@ Explore our tutorials to discover what you can do with |CL|!
...
    kata
-   kata_migration
    kubernetes/kubernetes*
    yubikey-u2f
    nvidia
    nvidia-cuda
    redis
-   tutorial-proxy
+   proxy
+   kata_migration
```
Comment (puneetse) [inline source/tutorials/index.rst]: Suggest ordering alphabetically to make it easier for readers to find if they know what they're looking for. 
Can we remove the `kata_migration` document altogether? See my note in the GH issue.

File: source/get-started/index.rst [hunk 6]
```diff
@@ -46,4 +46,13 @@ Install in a virtual machine
    :glob:
 
    virtual-machine-install/*
-   ../../guides/maintenance/increase-virtual-disk-size.rst
\ No newline at end of file
+   ../../guides/maintenance/increase-virtual-disk-size.rst
+
+Install in the cloud
```
Comment (puneetse) [inline source/get-started/index.rst]: I would recommend the phrase "Deploy" over "Install" when referring to cloud images. Same note regarding the navigation/path.

**Comment by intelkevinputnam** [comment]
@puneetse & @krisdale This is pretty near the state I want it to be in. Please take another look.

**Pull request merged**